schematic/models/commands.py

```diff
@@ -96,6 +96,13 @@ def model(ctx, config):  # use as `schematic model ...`
     is_flag=True,
     help=query_dict(model_commands, ("model", "validate", "restrict_rules")),
 )
+@click.option(
+    "--file_annotations_upload/--no-file_annotations_upload",
+    "-fa/--no-fa",
```

Comment:
`--no-fa` should be `-no-fa` to follow conventions :)

Reply:
Thans @mialy-defelice. I just updated that.